mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml

```diff
@@ -333,5 +333,12 @@
         <Required>N</Required>
       </exceptions>
     </surbl>
+    
```

Comment:
extra line

Reply:
I do have them everywhere between the sections in the model (every tab is such an block in the Model).
It makes the file easier to read.

Reply:
ok, fine